"""empty message

Revision ID: dacaa4333eee
Revises: ca7744bd2e6d
Create Date: 2017-08-23 13:18:25.246685

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = 'dacaa4333eee'
down_revision = 'ca7744bd2e6d'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###


    op.drop_column('auth_influencer', 'user_id')
    op.add_column('auth_influencer',
    sa.Column('stars', sa.Integer(), autoincrement=False, nullable=True)
)
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table('auth_user',
    sa.Column('id', sa.VARCHAR(length=64), autoincrement=False, nullable=False),
    sa.Column('first', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('last', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('company', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('website', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('email', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('phone', sa.VARCHAR(length=12), autoincrement=False, nullable=True),
    sa.Column('password', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.PrimaryKeyConstraint('id', name='auth_user_pkey'),
    sa.UniqueConstraint('email', name='auth_user_email_key'),
    postgresql_ignore_search_path=False
    )
    op.create_table('auth_influencer',
    sa.Column('id', sa.VARCHAR(length=64), autoincrement=False, nullable=False),
    sa.Column('handle', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('token', sa.VARCHAR(length=128), autoincrement=False, nullable=True),
    sa.Column('user_id', sa.VARCHAR(length=64), autoincrement=False, nullable=True),
    sa.ForeignKeyConstraint(['user_id'], ['auth_user.id'], name='auth_influencer_user_id_fkey'),
    sa.PrimaryKeyConstraint('id', name='auth_influencer_pkey'),
    postgresql_ignore_search_path=False
    )
    op.create_table('association',
    sa.Column('user_id', sa.VARCHAR(length=64), autoincrement=False, nullable=True),
    sa.Column('influencer_id', sa.VARCHAR(length=64), autoincrement=False, nullable=True),
    sa.ForeignKeyConstraint(['influencer_id'], ['auth_influencer.id'], name='association_influencer_id_fkey'),
    sa.ForeignKeyConstraint(['user_id'], ['auth_user.id'], name='association_user_id_fkey')
    )
    # ### end Alembic commands ###
